lib/portlayer/exec/base.go

```diff
@@ -300,9 +300,15 @@ func (c *containerBase) poweroff(ctx context.Context) error {
 			case *types.GenericVmConfigFault:
 
 				// Check if the poweroff task was canceled due to a concurrent guest shutdown
-				if len(terr.FaultMessage) > 0 && terr.FaultMessage[0].Key == vmNotSuspendedKey {
-					log.Infof("power off %s task skipped due to guest shutdown", c.ExecConfig.ID)
-					return nil
+				if len(terr.FaultMessage) > 0 {
+					switch terr.FaultMessage[0].Key {
+					case vmNotSuspendedKey:
+						fallthrough
```

Comment:
If we intend to do the same thing for both these cases, we can say `case vmNotSuspendedKey, vmPoweringOffKey:` instead of `fallthrough`. Talked about it with @matthewavery too, he also agrees the former is more readable.